assets/javascripts/modules/checkout/validatePayment.js

```diff
@@ -1,3 +1,4 @@
+/* global Raven */
```

Comment:
Raven is an error reporter (it's the client for [Sentry](https://app.getsentry.com/the-guardian/), so is a bit unique in that it *needs* to be present globally as it hooks into `window.onerror`  and various other things and runs outside the application code.